Sources/SmileID/Classes/BiometricKYC/OrchestratedBiometricKycViewModel.swift

```diff
@@ -38,7 +33,7 @@ class OrchestratedBiometricKycViewModel: ObservableObject {
         if selfieFile != nil {
             submitJob()
         } else {
-            updateStep(.selfie)
+            updateProcessingState(nil)
```

Comment:
We should maybe have a state that is not `nil`

Like use an actual name so it's even easier to go through the code and understand

Reply:
The state is used to drive navigation to the ProcessingScreen. nil means the job submission hasn't started, so processing screen isn't showing. That's why it's an optional value.

Reply:
I'd agree - we should have clear defined states. An optional seems a bit of a hack. Maybe we can call it `unSubmitted` or `notStarted`, or `init`?